src/future-hopr-lib-components/Layout/index.tsx

```diff
@@ -70,6 +70,7 @@ const Layout: React.FC<{
   };
   drawerType?: 'blue' | 'white';
   drawerItems: ApplicationMapType;
+  drawerFunctionItems? : any;
```

Comment:
```suggestion
  drawerFunctionItems? : ApplicationMapType;
```

Reply:
I think this is the type you use for this throughout the app right ?

Reply:
Fixed